subsys/spm/secure_services.c

```diff
@@ -68,6 +76,8 @@ int spm_request_read(void *destination, u32_t addr, size_t len)
 		{.start = PM_MCUBOOT_PAD_ADDRESS,
 		 .size = PM_MCUBOOT_PAD_SIZE},
 #endif
+		{FICR_PUBLIC_ADDR, FICR_PUBLIC_SIZE},
```

Comment:
can you use the same style for the instantiation here, as above?

Reply:
Yes, will do.